public/coda/styles.css

```diff
@@ -2,17 +2,50 @@
     background: #ffff;
     width: 100%;
     max-width:initial;
-    height:80px;
+    height:60px;
   }
 .nav-link {
     color: #8d3538; 
 }
 .brand-logo{
-  padding:5;
-  height:60px;
+  height:40px;
+  margin-top:10px;
+}
+a {
+    color: black;
 }
 .table-heading {
     background-color: #8e3436;
-    height: 50 px;
+    height: 30px;
     color:#ffff;
 }
+.coding-complete{
+    background-color: white;
```

Comment:
These codes are listed in the opposite order to the ones int he JS file. could you reorder for consistency and easy matching?

Reply:
Fixed